src/xtgeo/surface/regular_surface.py

```diff
@@ -534,6 +534,12 @@ def methods(cls):
 
         return txt
 
+    @deprecation.deprecated(
+        deprecated_in="1.16",
+        removed_in="3.0",
+        current_version=xtgeo.version,
+        details="method 'ensure_correct_values' is obsolute and will removed soon",
```

Comment:
Typo: obsolete, I guess this is not replaced by anything user facing?

Reply:
I don't think so, no.